eng/pipelines/tools.yml

```diff
@@ -0,0 +1,83 @@
+trigger:
+  branches:
+    include:
+    - users/mitchdenny/pipelines-evolution
+
+pr:
+  branches:
+    include:
+    - master
+
+jobs:
+- template: templates/jobs/archetype-tools-net.yml
+  parameters:
+    JobName: pixel_server
+    ProjectFile: tools/pixel-server/PixelServer.sln
+- template: templates/jobs/archetype-tools-net.yml
+  parameters:
+    JobName: roslyn_sdk_analyzers
+    ProjectFile: tools/roslyn-sdk-analyzers/Azure.ClientSdk.Analyzers/Azure.ClientSdk.Analyzers.sln
+- job: 'build_doc_warden'
+  displayName: 'Building and Testing: tools/doc-warden'
+  pool:
```

Comment:
Why not turn the python builds into a template as well?

Reply:
Looks like you do have the warden template but didn't use it.

Reply:
I'm leaving the existing pipeline files there until I'm done, I'll probably remove them as a final step. For the Python stuff I was waiting to see if a template was required. I could turn this into a job template but I was hesitant to do that until I get another fully defined Python module like doc-warden to extract the common pattern.

The other python scripts aren't fully defined modules so their build process is a little different.

Reply:
Even if there is only one tool it seems like it might be cleaner to have a template per job type and have the tools.yml be small and simple with most of the complexity and heavy lifting in the job type template.

Reply:
Fair enough - I'll break it how into its own template.